ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml

```diff
@@ -97,6 +97,10 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-disruption
+- as: e2e-gcp-disruptive-ovn
```

Comment:
Please add job for aws as well (also optional)

Reply:
both are optional now